receiver/elasticapmreceiver/testdata/invalid_ids_expected.yaml

```diff
@@ -55,7 +55,7 @@ resourceSpans:
                   stringValue: ""
               - key: transaction.duration.us
                 value:
-                  intValue: "32592981"
+                  intValue: "32592"
```

Comment:
Note there is no real data loss here.

`startTimeUnixNano` and `endTimeUnixNano` are unchanged - this is only for this attribute to satisfy Kibana requirements. Same for other instances in the `yaml` files.